test/Microsoft.DotNet.Tools.Tests.Utilities/TestBase.cs

```diff
@@ -99,76 +99,5 @@ private bool PreserveTemp()
                 string.Equals("1", val, StringComparison.OrdinalIgnoreCase) ||
                 string.Equals("on", val, StringComparison.OrdinalIgnoreCase));
         }
-
-        protected CommandResult TestExecutable(string outputDir,
```

Comment:
dead code that depend on system.text.json but is netstandard

Reply:
This seems ok to me but I don't have much context here.